apps/web/src/components/agent-inbox/hooks/use-interrupted-actions.tsx

```diff
@@ -313,6 +314,76 @@ export default function useInterruptedActions<
     setLoading(false);
   };
 
+  const handleScheduledSubmit = async (scheduledTime: Date) => {
+    if (!agentInboxId) {
+      toast.error("No agent inbox ID found");
```

Comment:
nit, without this the toast will be black and white. add to others too plz
```suggestion
      toast.error("No agent inbox ID found", { richColors: true });
```